TrackingTools/TransientTrack/src/TransientTrackBuilder.cc

```diff
@@ -106,7 +106,7 @@ TransientTrackBuilder::build ( const edm::Handle<reco::TrackCollection> & trkCol
     TrackRef ref(trkColl, i);
     double time = trackTimes[ref];
     double timeReso = trackTimeResos[ref];
-    timeReso = ( timeReso > 1e-6 ? timeReso : fakeBeamSpotTimeWidth ); // make the error much larger than the BS time width
+    timeReso = ( timeReso > 1e-6 ? timeReso : 2.0*fakeBeamSpotTimeWidth ); // make the error much larger than the BS time width
```

Comment:
I suppose, the ```2.0*fakeBeamSpotTimeWidth ``` here is made to be in sync with cases of invalid times in trackTimes (a few lines below).
Is this correct? 
Given repeated multiplication by 2.0, perhaps it's better to define smth like ```defaultInvalidTrackReso = 2.0*fakeBeamSpotTimeWidth``` .

Also, I see that the hardcoded magic value of fakeBeamSpotTimeWidth is 0.175f in this file.  It previously  was in sync with the value in TrackTimeValueMapProducer.cc . Now they have diverged.

Reply:
Yes the one in the valuemap producer is no longer used as said, and this one is really just a default error which need be only vaguely similar to the beamspot width.

Reply:
Yes the one in the valuemap producer is no longer used as said, and this one is really just a default error which need be only vaguely similar to the beamspot width.